Metagenomics/docker/Dockerfile

```diff
@@ -1,30 +1,12 @@
-# Galaxy - <topic name> training material
+# Galaxy - 16S-Metagenomics
 
-FROM <base-galaxy-flavor>
+FROM bgruening/galaxy-ngs-preprocessing
 
-MAINTAINER <maintainer-name>, <maintainer-email>
+MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com
 
-ENV GALAXY_CONFIG_BRAND Topic Name
+ENV GALAXY_CONFIG_BRAND "Galaxy 16S Mothur SOP"
 ENV ENABLE_TTS_INSTALL True
```

Comment:
The enable TTS install can also be removed. We only install tools from the Main ToolShed.